### ISSUE TITLE
[feature request][kubernetes] Add Company filter for the following charts

### ISSUE THREAD

**Issue by mrbobbytables** [cncf/devstats.archive#295]
Follow up from #289

**NOTE:** This is it for now, I need to think a bit on some of the other dashboards on how best to represent company involvement.

---

## Problem 3 - Companies cannot reasonably measure their involvement

**Feature Request:** On dashboards where `repo_group` is present, allow further filtering by Company.

As a company trying to gauge our involvement within a particular SIG or subproject, we cannot establish what areas we are contributing to because `repo_group` does not provide enough detail (the previous two requests).

Being able to look at the areas, and check company investment helps highlight areas that need potential headcount, or internal re-alignment to ensure the upstream group/repo/subproject remains healthy.

(No examples, please see above requests examples, but think of company metric)

[kubernetes/kubernetes]: https://github.com/kubernetes/kubernetes
[kubernetes/enhancements]: https://github.com/kubernetes/enhancements

---

## Dashboards

- Community Sizing and Health Assessment Dashboards
  - GitHub Stats by Repository
  - GitHub Stats by Repository Groups

**Comment by lukaszgryglicki** [comment]
On it now.

**Comment by lukaszgryglicki** [comment]
Not quite sure what changs should be added to those two dashes, see you have [Company activity by repos/repo groups](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1), if I add company drop-down there, they wil just become this one.
If you really need this, please specify EXACTLY:
- What drop-downs are needed (repo?, repo group? company? contribution type?)
- Multi-value dropdowns? Single-value?
- What data should be displayed on charts? Single value, Stacked companies data? Stacked repo/repo group data? Something else?

I cannot proceed now, because I think you already have that data.

**Comment by mrbobbytables** [comment]
That was my bad, I did completely miss that chart when reviewing them the other day :(

TL;DR I think we can close this, small rant ahead

It is actually a harder than I first anticipated. Initially when looking at all the dashboards there was a significant amount of boards that would be beneficial for company data, but then REALLY thinking about how to get meaningful or actionable data out of it, it'd be much harder. e.g. trying to determine if a company is falling behind on reviews and looking at total vs open, but thats not a good indicator because of how reviewers are assigned.

The one place where I think we can get meaningful data out of it is the contributions chart, and adding filter for `sig/*` and `wg/*` labels.

As a company I can then easily our involvement in each sig/wg for enhancements, k/k etc. This is the same for SIG leads and would solve the issue for [sig-security](https://github.com/cncf/devstats/issues/293) since they're largely jut issue based and working out of k/community and would help a few other areas alluded to in my last comment here: https://github.com/cncf/devstats/issues/289#issuecomment-846631395

**Comment by mrbobbytables** [comment]
closing in favor of #299
